R/gh_fill.R

```diff
@@ -0,0 +1,14 @@
+gh_fill <- function(geohashes, precision) {
+  if (length(unique(nchar(geohashes))) > 1) {
+    stop("Input Geohashes must all have the same precision level.")
```

Comment:
Is there no use case for taking e.g. `c("0", "11")` as input and wanting all of the level-4 precisions as output?